src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -33,7 +33,8 @@
 					<EmoticonOutline :size="20" />
 				</template>
 			</NcButton>
-			<NcActions v-show="isReplyable">
+			<!-- TODO change with NcButton and adjust tests-->
+			<NcActions v-show="!isConversationReadOnly && isReplyable">
```

Comment:
https://github.com/nextcloud/spreed/blob/master/lib/Model/Message.php#L159

hmm should already be the case?

Reply:
They come from different store, and only one of them changes when toggling lock/unlock.
One more reason to refetch...